src/main/java/at/hannibal2/skyhanni/features/inventory/chocolatefactory/ChocolateFactoryDataLoader.kt

```diff
@@ -280,8 +282,15 @@ object ChocolateFactoryDataLoader {
 
     private fun processItem(list: MutableList<ChocolateFactoryUpgrade>, item: ItemStack, slotIndex: Int) {
         if (slotIndex == ChocolateFactoryAPI.prestigeIndex) return
-        if (config.rabbitWarning && clickMeRabbitPattern.matches(item.name)) {
-            SoundUtils.playBeepSound()
+        if (clickMeRabbitPattern.matches(item.name)) {
```

Comment:
Since this is inside the "data loader" class: Can we instead use a "cf menu update event" or simply use an independent inventory open event (we don't need cf info for this check) 
not 100% needed for this pr as the original logic was also here, just one thought for the future of this class.

Reply:
Since i'm only fixing sound for Golden rabbits and adding collection rabbits you need to click, i'll save that for later